README.rst

```diff
@@ -6,59 +6,43 @@ Taisei
 Introduction
 ------------
 
-Taisei is an open clone of the Tōhō Project series. Tōhō is a one-man project of
-shoot-em-up games set in an isolated world full of Japanese folklore.
+Taisei Project is an open clone of the Tōhō Project series. Tōhō is a one-man
```

Comment:
I know it says "clone" in the original as well, but I think it's a good opportunity to correct that - we should emphasize that it is a fangame, and not an attempt to recreate any of ZUN's games.

It has been unfortunately misconstrued in the past ([here is one of the most painful examples](https://community.chrono.gg/t/taisei-bullet-hell-madness/11065)). That's why our website even links to the wikipedia page on fangames.